jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -45,7 +45,7 @@ public static SchemaMetadata readRemoteMetadata(final String connectionString, f
 
             schema = findSchema(schema, dbMeta, dialect);
 
-            final List<TableMetadata> tables = findTables(catalog, schema, tableFilter, dbMeta, dialect, exceptionMode);
+            final List<TableMetadata> tables = findTables(catalog, schema, tableFilter, dbMeta, dialect, exceptionMode, ignoreErrorList);
 
             conn.close();
```

Comment:
Safety: `java.sql.Connection` implements `AutoClosable`. Use try-with-resources and remove manual `close()`.

Reply:
done